### DISCUSSION TITLE
Accedere all'applicazione home assistant in locale con DuckDNS

### DISCUSSION THREAD

**Discussion by davide1295** [indomus/forum#1634] [HUB: Home Assistant]
Ciao a tutti, ho configurato correttamente il certificato ed il dominio DuckDNS e riesco ad accedere al sistema tramite gli indirizzi _https://miodominio.duckdns.org_ e _https://192.168.xx.xx:8123_. Il problema sorge quando faccio l'accesso dall'applicazione per IOS utilizzando il link _https://192.168.xx.xx:8123_, ricevo l'errore "certificato non valido, NSURLErrorDomain - 1202.
Come potrei risolvere?

**Comment by indomus** [comment]
Non puoi: è normale che ti dia quell'errore. La soluzione è collegarsi via http:// e non https://. 
Che configurazione hai usato per definire l'ssl?

**Reply by davide1295** [reply]
Ho seguito passo passo questa guida: [https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-con-distribuzione-hassio/](url)
Se provo a collegarmi in http (_http://192.168.xx.xx:8123_) mi compare un errore e non riesco ad accedere.

**Reply by indomus** [reply]
Ok. I casi dunque sono tre:

- ti colleghi via fqnd anche dall'interno;
- ti colleghi via https con l'indirizzo locale e ti tieni l'errore;
- installi [reverse proxy](https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reverse-proxy/), il che ti consente di collegarti via http in locale e in https da remoto.
